### PR TITLE
Use ':' for fuzz target project qualified name separator if project name is a path

### PR DESCRIPTION
**Pull request by inferno-chromium**
path.

### CODE REVIEW

File: src/python/datastore/data_types.py [hunk 1]
```diff
@@ -1063,7 +1063,8 @@ def fuzz_target_project_qualified_name(project, binary):
     # Don't prefix with project name if it's the default project.
     return binary
 
-  project_prefix = project + '_'
+  seperator = '/' if project.startswith('/') else '_'
```
Comment (oliverchang) [inline src/python/datastore/data_types.py]: nit: correct spelling is "separator"

Comment (inferno-chromium) [reply src/python/datastore/data_types.py]: Done!

**Pull request merged**